### DISCUSSION TITLE
Lesson 14 - All imports got the error : File import callback not supported

### DISCUSSION THREAD

**Discussion by Web3Jumb0** [Cyfrin/foundry-full-course-cu#2079] [Q&A]
Hi mates, I have been struggling for the whole night and not sure why suddenly all the files in my src folder with the imports shows the same error 

```solidity
// SPDX-License-Identifier: MIT

pragma solidity ^0.8.19;

import {TimelockController} from "@openzeppelin/contracts/governance/TimelockController.sol";

contract TimeLock is TimelockController {
    constructor(uint256 minDelay, address[] memory proposers, address[] memory executors)
        TimelockController(minDelay, proposers, executors, msg.sender)
    {}
}
```

the import line will have a underline with the error:
`Source "@openzeppelin/contracts/governance/TimelockController.sol" not found: File import callback not supported(6275)`

This happens to the rest of the files...
Not sure what goes wrong... Orz

**Comment by EngrPips** [comment]
Hello @Web3Jumb0, This seems weird, but have you checked correctly that the import paths are correct? I will normally use `command + p` to search for the file I want to import and use the path I get by searching for the file as my import path.

**Reply by Web3Jumb0** [reply]
Thanks man.
I find out that there is something wrong with my tmol file...I typoe `remappings` to `remapptin**p**s`
The funny things is that the `command+left click` still work.

**Reply by EngrPips** [reply]
Interesting. Keep crushing it.
